fix: added automated tests to test video input, cleaned up workflow (should hopefully work)

gould-ann opened the pull request.

gould-ann committed:
added automated tests to test video input

katebrune commented on tests/script_tests.py:
```diff
@@ -0,0 +1,22 @@
+        pass
+
+    # all tests must start with "test"
+    def test_headless_encoding(self):
```
![image](https://user-images.githubusercontent.com/31379789/74561607-84017780-4f2e-11ea-983e-380e86c24345.png)

not sure why but this is where the workflow failed

gould-ann replied:
I think it's because the workflow just runs the tests... Like we probably need to additionally import our image files into the workflow itself in order to get that to work :^)

gould-ann committed:
final modifications

gould-ann committed:
should work now

final modifications

final modifications

final modifications

removed tests, changed to work locally

removed tests, changed to work locally??? where the fuck did my last commit go lol

gould-ann committed:
ready to merge
